subsystems/dvb/Makefile

```diff
@@ -0,0 +1,26 @@
+SPECFILE_SUBPACKAGE_DVB=rpm/dvb/dvb.spec
+
+.PHONY: dist
+dist: ##             - Creates the QM dvb package
+	cd $(ROOTDIR) && tar cvz \
+		--exclude-from=build-aux/exclude_from_tar_gz_subpackage_kvm.txt \
+		--dereference \
+		--transform s/qm/qm-dvb-${VERSION}/ \
```

Comment:
why dvb now?

Reply:
I was running all rpm/dir/*.spec file
It was there, already 
https://github.com/containers/qm/tree/main/rpm/dvb